inc/osvr/ClientKit/Display.h

```diff
@@ -155,6 +155,21 @@ namespace clientkit {
             }
         }
 
+        /// @brief Gets the clipping planes.
+        ///
+        /// @sa osvrClientGetViewerEyeSurfaceProjectionClippingPlanes
+        void getProjectionClippingPlanes(double *left, double *right,
+                                         double *bottom, double *top) const {
```

Comment:
In the C++ API, if I can't just return a return value, the pattern is to take parameters by reference (and take their address inside the wrapper) - more C++-like.